lib/db/entity/activity_progress.dart

```diff
@@ -0,0 +1,48 @@
+class ActivityProgress {
+  static const table = 'activityProgress';
+  static const userIdCol = 'userId';
+  static const topicIdCol = 'topicId';
+  static const activityIdCol = 'activityId';
+
+  String userId;
+  String topicId;
+  String activityId;
```

Comment:
Can you add a timestamp as well to track when the user attempted this activity?

Reply:
Yeah, Sure, Done!